src/chunk.c

```diff
@@ -1467,12 +1467,7 @@ chunk_get_chunks_in_time_range(Oid table_relid, Datum older_than_datum, Datum ne
 	}
 	else
 	{
-		ht = ts_hypertable_cache_get_entry(hypertable_cache, table_relid);
-		if (!ht)
-			ereport(ERROR,
-					(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
-					 errmsg("table \"%s\" does not exist or is not a hypertable",
-							get_rel_name(table_relid))));
+		ht = ts_hypertable_cache_get_entry_always(table_relid, hypertable_cache);
```

Comment:
Find `_always` a bit ambiguous. I suggest `_or_fail` or similar.

Otherwise, a common pattern in PostgreSQL and also our own code is to have a boolean parameter called `missing_ok` or similar, which determines the behavior of the function. I guess this could be wrapped by a macro or other function that take only two arguments.